app/views/entities/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<% @title = "Entities" %>
+  <%= render "shared/navbar", locals: { title: @title }, back: true %>
+
+<div class="mb-10">
+
+  <% if @entities.empty?%>
+    <p class="text-center pt-16 mb-10">Currently you don't have any entity</p>
+  <% else %>
+    <ul class="list-group mb-5 list-group-flush pt-12">
+      <li class="list-group-item align-items-center gap-3 px-3 py-5 my-5 bg-white">
+          <div class="d-flex justify-content-start gap-3 align-items-center">
+            <figure class="image"><img class="rounded mx-auto d-block" src="<%= @group.icon %>" alt="user icon""></figure>
+            <div class="text-lg opacity-75 font-bold text-uppercase"><%= @group.name %></div>
+          </div>
+      </li>
+      <% @entities.each do |entity| %>
+        <li class="entity-card list-group-item mb-3 px-3 py-5 bg-white">
+          <div class="d-flex justify-content-between align-items-center p-2">
+              <span class="capitalize text-gray-500 font-semibold"><%= entity.name %> - </span>
+              <span class="font-semibold text-lg"> $<%= number_with_delimiter(entity.amount)%></span>
+          </div>
+          <div class="d-flex">
+              <span class="capitalize text-gray-300 font-light"><%= entity.created_at.strftime("%d %b, %y") %></span>
+          </div>
+        </li>
+      <% end %>
+      <li class="list-group-item align-items-center gap-3 px-3 py-5 my-5 bg-white">
+          <div class='d-flex justify-content-start align-items-center'>
+            <hr>
+          <div class="d-flex justify-content-center mb-4 group-text">
+              <div class="text-secondary font-dark-sm"><%= "TOTAL PAYMENT #{ number_to_currency(number_with_delimiter(@total)) }" %></div>
+          </div>
+      </li>
+    </ul>
+  <% end %>
+
+  <%= link_to "ADD NEW ENTITY", new_group_entity_path(@group), class: 'fixed-bottom border border-white text-center text-white py-3 w-full', style: 'background-color: #5fb523; padding: 1rem 0' %>
```

Comment:
- Hey, kindly remove the text decoration of the underline on this link, this is a button, so it is not a great user experience having the text underlined.


*your design*
![image](https://github.com/Bhekisile/budget-app/assets/59397159/ea056c47-3091-41cb-9d1c-4b3bf3f331a1)